app.py

```diff
@@ -27,6 +28,8 @@ def create_app(config_object=None):
             traces_sample_rate=1.0
         )
 
+    babel = Babel(app)
```

Comment:
don't assign babel to a local variable as you're not using it afterwards, also the following newline is not needed.
Take a lookat how `CORS(app)` is initialized. You have the exact same usecase.

Reply:
On row 57 of `app.py` the `babel` local variable is used in a decorator to define the `get_locale` function for the locale selector.

Reply:
Ok, I see. flask-babel has the init_app interface like many other flask extensions. I thought you should use it:
https://flask-babel.tkte.ch/#flask_babel.Babel.init_app
and configure it like we do with db. Then you can have all babel configuration done in a separate file.
But it'll have the same problem of accessing app.config before creating the flask app (a problem we've discussed in nav engine and sql-alchemy localization).

It's not ideal but let's leave it like that. Thanks.